docs/app/services/version/version.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable, Output, EventEmitter } from '@angular/core';
+import { BehaviorSubject } from 'rxjs/BehaviorSubject';
+
+
+@Injectable()
+export class VersionService {
+
+    version: BehaviorSubject<Version> = new BehaviorSubject<Version>(Version.Angular);
+
+    constructor() {
+        this.toggle(this.toVersion(window.localStorage.getItem('version')));
```

Comment:
If there is no previously stored version in local storage this toVersion call breaks the application. Need to add a check in `toVersion` to handle when version is empty